app/client/lib/helpers/helperFunctions.js

```diff
@@ -258,7 +258,7 @@ Helpers.formatTransactionBalance = function(value, exchangeRates, unit) {
         unit = null;
 
     var unit = unit || EthTools.getUnit(),
-        format = '0,0.00';
+        format = '0,0.00[0]';
```

Comment:
Thats will break the addition of bitcoin optionals at line 268.
Why you want to add this one optional here?